CRVResponse/test/calibration.fcl

```diff
@@ -1,103 +1,75 @@
 #
-# generate and simulate full CRV waveforms to find the pedestal
+# generate and simulate full CRV waveforms to reconstruct dark counts
 #
 #include "Offline/fcl/standardServices.fcl"
 #include "Offline/EventGenerator/fcl/prolog.fcl"
 #include "Production/JobConfig/common/prolog.fcl"
 #include "Production/JobConfig/primary/prolog.fcl"
 #include "Production/JobConfig/digitize/prolog.fcl"
 #include "Production/JobConfig/cosmic/prolog.fcl"
-#include "Production/JobConfig/reco/prolog.fcl"
 
-process_name : cosmicSimReco
+process_name : crvCalibSim
 
 source : {
   module_type :EmptyEvent
   firstRun: 1200
 }
-services : @local::Services.Sim
+services: @local::Services.SimAndReco
 
 physics: {
   producers : {
+    @table::CommonMC.producers
     @table::Common.producers
     @table::Primary.producers
-    @table::Digitize.producers
-    @table::Reconstruction.producers
     cosmicTimeOffset : @local::Cosmic.cosmicTimeOffset
-    generate : @local::Cosmic.generateCRY
+    @table::CrvDAQPackage.producers
     CrvRecoPulses: @local::CrvRecoPulses
   }
-  analyzers: {
-    CrvCalibration: @local::CrvCalibration
+  filters : {
+    @table::Common.filters
   }
 
-  TriggerPath : [
-    @sequence::Common.generateSequence,
-    cosmicTimeOffset,
-    g4run,
-    CrvSteps,
-    EWMProducer,
-    @sequence::CrvDAQPackage.CrvDAQSequence,
-    CrvRecoPulses
-  ]
-  EndPath : [ CrvCalibration ]
+  TriggerPath :  [ @sequence::Common.generateSequence, cosmicTimeOffset, @sequence::Common.g4Sequence, @sequence::CommonMC.DigiSim,
+                   CrvSteps, @sequence::CrvDAQPackage.CrvDAQSequence, CrvRecoPulses ]
+  out: [Output]
   trigger_paths : [ TriggerPath ]
-  end_paths : [ EndPath ]
+  end_paths : [ out ]
 }
 
-#include "Production/JobConfig/reco/epilog.fcl"
-physics.analyzers.digiCompressionCheck.SelectEvents : [ TriggerPath ]
-physics.producers.FindMCPrimary.PrimaryGenIds: ["CosmicCRY"]
-
-# set the event timing for OffSpill
-physics.producers.EWMProducer.SpillType : 0
-# override the Untriggered filter to just prescale
-# nofield geometry
-services.GeometryService.bFieldFile : "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
-# print summaries
-services.scheduler.wantSummary: true
-services.TimeTracker.printSummary: true
+outputs:
+{
+  Output :
+  {
+    module_type : RootOutput
+    fileName    : "crv_calib.art"
+    outputCommands: ["keep *",
+                     "drop *SimParticle*_*_*_*",
+                     "drop *StepPointMC*_*_*_*",
+                     "drop *CrvStep*_*_*_*",
+                     "drop *CrvPhotons*_*_*_*",
+                     "drop *CrvSiPMCharges*_*_*_*",
+                     "drop *CrvDigiMC*_*_*_*"]
+  }
+}
 
-# g4 config
-physics.producers.g4run.TrajectoryControl: @local::mu2eg4NoTrajectories
-physics.producers.g4run.inputs.primaryTag : cosmicTimeOffset
-physics.producers.g4run.Mu2eG4CommonCut: @local::Cosmic.Mu2eG4CommonCutCosmicNoField
-physics.producers.g4run.physics.minRangeCut : 1.0 // mm
```

Comment:
I would keep this cut. It makes a significant impact on the file size.

Reply:
True. I will bring them back.